roles/consul/tasks/main.yml

```diff
@@ -39,8 +39,8 @@
 
 - name: setup TCP iptables for consul
   shell: >
-      ( iptables -L INPUT | grep -q 'consul tcp ({{ item }})' ) || \
-      iptables -I INPUT 1 -p tcp --dport {{ item }} -j ACCEPT -m comment --comment "consul tcp ({{ item }})" 
+      ( iptables -L INPUT | grep -q 'contiv_consul tcp ({{ item }})' ) || \
+      iptables -I INPUT 1 -p tcp --dport {{ item }} -j ACCEPT -m comment --comment "contiv_consul tcp ({{ item }})" 
```

Comment:
there is `iptables --check` in rhel 7.4 that helps you check if a rule exists

Reply:
Does --check avaliable to all platform?

Reply:
not all only exists on newer distribution

Reply:
I  just changed the tag name in existing rule code. Don't want  to change current logic for iptables. Also this current logic  would work for all OS distribution. 
We can open new PR for your suggested approach and fix after CVD . This approach require to change in all available rules.